### DISCUSSION TITLE
[Request - Generic] Reduce Resoucepack size

### DISCUSSION THREAD

**Discussion by TheFloodDragon** [xenondevs/Nova#401] [Q&A] [answered]
### Describe your feature request.

Nova seems to pack a vanilla resource pack in the output resource pack, making the pack large (More than 10MB).

Are the Minecraft vanilla assets necessary? (In case I have disabled waila)

And could we choose some languages instead of all languages packed in the output resource pack?

### Additional context

Hope for your reply!

**Comment by NichtStudioCode** [comment]
> Nova seems to pack a vanilla resource pack in the output resource pack

It doesn't

> Are the Minecraft vanilla assets necessary? (In case I have disabled waila)

You may also want to disable boss bar overlay then (assuming no other addon on your server uses it) to reduce the size of the resource pack.

> And could we choose some languages instead of all languages packed in the output resource pack?

See https://docs.xen.cx/nova/admin/configuration/#resource-filters

**Reply by TheFloodDragon** [reply]
Oh, thanks for your reply.

I revisited the Nova resource pack and noticed, that font files take up most of the total size. (About 3/4, nearly 12MB)

Here are the paths: `assets\minecraft\font\include\unifont` `assets\minecraft\textures\font\include\unifont\nova_bmp`

I'm thinking if you can work on these, the size will be smaller well.

**Reply by NichtStudioCode** [reply, marked as answer]
I am aware of that. Those should be excluded if you disable boss bar overlay.

**Reply by TheFloodDragon** [reply]
Ok, thanks.
